privacyidea/models.py

```diff
@@ -693,6 +693,65 @@ def save(self, persistent=True):
         return ret
 
 
+class CustomUserAttribute(MethodsMixin, db.Model):
+    """
+    The table "customuserattribute" is used to store additional, internal attributes
+    for also external users.
```

Comment:
what was the *external user* again? Since the internal users (aka admins) can't have custom attributes (they do not have a resolver) maybe we should stick to (regular) user?

Reply:
Done in d028f89c